Makefile

```diff
@@ -1,18 +1,26 @@
-TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
+TOOL_BIN := bin/gotools/$(shell uname -s)-$(shell uname -m)
+PROTO_FILES := $(shell find proto/ -type f -name '*.proto')
 
-PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):`pwd`/node_modules/.bin:${PATH}"
+export GOBIN := $(shell pwd)/$(TOOL_BIN)
+export PATH := $(GOBIN):$(shell npm root)/.bin:$(PATH)
 
-PROTO_FILES=$(shell find proto/ -type f -name '*.proto')
+.PHONY: all
```

Comment:
We were missing `PHONY` on all our phony targets

Reply:
As discussed in Slack, leaving these as per-target `.PHONY`'s as recommended by [this Makefile styleguide](https://clarkgrubb.com/makefile-style-guide) to make individual phony targets easier to grok/modify/remove